Automattic/jetpack
# Subscribe block: Hide social followers toggle when showSubscribersTotal is not on or isPublicizeEnabled is false

Fixes #28725

## Proposed changes:
This PR hides the "Include social followers in count" when "Show subscribers count" is off, or when Publicize is not enabled.

<img width="272" alt="CleanShot 2023-02-14 at 15 06 59@2x" src="https://user-images.githubusercontent.com/528287/218761762-ef6c4116-51af-4796-9480-a782f107b127.png">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:
- Apply this PR
- Create a new post
- Add a Subscribe block
- "Include social followers in count" should be hidden when "Show subscribers count" is off, or Publicize is disabled.
